runtime/runtime/src/verifier.rs

```diff
@@ -1736,18 +2042,31 @@ mod tests {
     }
 
     #[test]
-    fn test_validate_transaction_exceeding_tx_size_limit() {
+    fn test_validate_transaction_deposit_with_function_call() {
+        test_validate_transaction_deposit_with_function_call_impl(SignerKind::AccessKey);
+        test_validate_transaction_deposit_with_function_call_impl(SignerKind::GasKey(0));
```

Comment:
those should be 2 different tests:
test_validate_transaction_deposit_with_function_call_access_key
test_validate_transaction_deposit_with_function_call_gas_key